fish_prompt.fish

```diff
@@ -1,3 +1,8 @@
+# Set defaults
+set -q SPACEFISH_CHAR_PREFIX ""
+set -q SPACEFISH_CHAR_SYMBOL ➜
+set -q SPACEFISH_CHAR_SUFFIX " "
```

Comment:
I've created the [`__sf_util_set_default`](https://github.com/matchai/spacefish/blob/master/functions/__sf_util_set_default.fish) fish function that allows you to not override user configs in config.fish.
Could you please swap out these `set -q` with it?